shared/src/validation/master/master-sign-up/master-sign-up.validation-schema.ts

```diff
@@ -12,8 +12,24 @@ const masterSignUp = Joi.object({
       'string.email': MasterSignUpValidationMessage.EMAIL_WRONG,
       'string.empty': MasterSignUpValidationMessage.EMAIL_REQUIRE,
     }),
-  [getNameOf<MasterSignUpDto>('name')]: Joi.string().trim().required(),
-  [getNameOf<MasterSignUpDto>('password')]: Joi.string().trim().required(),
+  [getNameOf<MasterSignUpDto>('name')]: Joi.string()
+    .trim()
+    .min(4)
```

Comment:
```suggestion
    .min(MasterSignUpValidationRule.NAME_MIN_LENGTH)
```
avoid magic number. add it for all statements